core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts

```diff
@@ -536,6 +538,12 @@ angular.module('oppia').directive('audioTranslationBar', [
 
             $scope.$on('$destroy', function() {
               document.body.onkeyup = null;
+              // Remove jQuery event listeners.
+              $('.oppia-translation-tab').off('dragover');
+              $('.oppia-main-body').off('dragleave');
+              $('.oppia-translation-tab').off('drop');
```

Comment:
We are trying to get rid of jQuery in long term. Can you add a TODO here referring to #12146

Reply:
Got it, done!